common/core/desktop/src/kmx/kmx_file.cpp

```diff
@@ -18,6 +19,37 @@ KMX_BOOL KMX_ProcessEvent::Load(km_kbp_path_name KeyboardName)
   return TRUE;
 }
 
+const int km::kbp::kmx::CODE__SIZE[] = {
+    -1,  // undefined                0x00
+    1,   // CODE_ANY                 0x01
+    2,   // CODE_INDEX               0x02
+    0,   // CODE_CONTEXT             0x03
+    0,   // CODE_NUL                 0x04
+    1,   // CODE_USE                 0x05
+    0,   // CODE_RETURN              0x06
+    0,   // CODE_BEEP                0x07
+    1,   // CODE_DEADKEY             0x08
+    0,   // CODE_BACKSPACE           0x09
+    2,   // CODE_EXTENDED            0x0A
+    0,   // CODE_EXTENDEDEND         0x0B
+    0,   // CODE_SWITCH              0x0C
```

Comment:
```suggestion
    1,   // CODE_SWITCH              0x0C
```

Per https://github.com/keymanapp/keyman/blob/d1dd8b022e7323dc44339d52aabb5380299429e6/windows/src/developer/kmcmpdll/Compiler.cpp#L2383-L2386